src/layouts/MainLayout.vue

```diff
@@ -100,7 +101,14 @@ import { Vue, Component } from 'vue-property-decorator';
   components: { EssentialLink }
 })
 export default class MainLayout extends Vue {
-  leftDrawerOpen = false;
-  essentialLinks = linksData;
+  headerColor() {
+    if (this.$route.path == '/learn') {
```

Comment:
I'm seeing this condition at many parts of main layout... better to simplify this as 
```suggestion
    if (this.onLearnPage()) {
```

by having a method like
```js
onLearnPage() {
     retun this.$route.path.startsWith('/learn');
}
```